src/Manslice.java

```diff
@@ -0,0 +1,290 @@
+import java.util.Arrays;
+
+import components.queue.Queue;
+import components.queue.Queue2;
+import components.simplewriter.SimpleWriter;
+import components.simplewriter.SimpleWriter1L;
+
+public class Manslice {
+
+    private String chordVoice, melodyVoice, rhythmVoice, bassVoice;
+
+    private int BPM;
```

Comment:
By convention, we usually still use lowerCamelCase for variables and UPPERCASE for constants. 
```suggestion
    private int bpm;
```